src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
```

Comment:
Consider using SCSS variables for image paths to improve maintainability and consistency across your styles.